src/components/Table/Table.jsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import "./Table.css";
+
+const Table = ({ columns, headers, children }) => {
```

Comment:
Remove prop `columns` and try use default params

```suggestion
const Table = ({ headers = [], children }) => {
```